service-base/src/main/java/org/eclipse/hono/service/registration/RegistrationHttpEndpoint.java

```diff
@@ -12,6 +12,8 @@
 
 package org.eclipse.hono.service.registration;
 
+import static org.eclipse.hono.util.RequestResponseApiConstants.FIELD_DEVICE_ID;
```

Comment:
please do not statically import the constant
instead, just import the class and refer to the constant using the class as qualifier so that in the code it is easier to see where the constant is coming from